# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/dvhtool/hotfix-getopt.patch.musl
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- dvhtool-1.0.1.orig/getopt.h.vanilla	2025-07-24 21:26:05.529246333 +0200
+++ dvhtool-1.0.1.orig/getopt.h	2025-07-24 21:27:09.745454963 +0200
@@ -127,7 +127,7 @@
    `getopt'.  */
 
 #if defined __STDC__ && __STDC__
-# ifdef __GNU_LIBRARY__
+# if 1 // def __GNU_LIBRARY__
 /* Many other libraries have conflicting prototypes for getopt, with
    differences in the consts, in stdlib.h.  To avoid compilation
    errors, only prototype getopt for the GNU C library.  */
